runsc/container/container.go

```diff
@@ -521,6 +521,28 @@ func Run(conf *config.Config, args Args) (unix.WaitStatus, error) {
 			return 0, fmt.Errorf("starting container: %v", err)
 		}
 	}
+
+	// If we allocate a terminal, forward signals to the sandbox process.
+	// Otherwise, Ctrl+C will terminate this process and its children,
+	// including the terminal.
+	if c.Spec.Process.Terminal {
+		sighandling.StartSignalForwarding(func(signal linux.Signal) {
+			if signal == linux.SIGPIPE || signal == linux.SIGCHLD {
```

Comment:
Can you leave a comment explaining the exceptions for SIGPIPE and SIGCHILD ?

Reply:
I have moved `SIGCHLD` into the `StartSignalForwarding`, where `SIGPIPE` was already dealt with.

https://github.com/google/gvisor/blob/ceea0cb491f48387de3c2cee3ec21e477bfecacd/pkg/sighandling/sighandling.go#L100-L103

I also added a comment explaining those exceptions in general.
https://github.com/google/gvisor/blob/ceea0cb491f48387de3c2cee3ec21e477bfecacd/pkg/sighandling/sighandling.go#L85-L86